file_paths.md

```diff
@@ -121,14 +126,22 @@ The files annotated by an '*' indicate that they are created by package installa
         ssl *
         VERSION                           # puppet-agent package version
 
+    /opt/puppetlabs/pxp-agent *
+        modules *
+            puppet *                      # executes Puppet and provides results of Puppet runs
```

Comment:
Should we name this `puppet` or `pxp-module-puppet`? I have a mild leaning toward the latter, even though it has redundant elements (since we're in a pxp-agent modules directory), because it matches the repository name and makes it easier to `find` the module when one forgets where it landed.